scripts/index.js

```diff
@@ -0,0 +1,102 @@
+import { clock } from "./clock.js";
+import { renderItems } from "./renderItems.js";
+const startBtn = document.querySelector(".start");
+const menuDisplay = document.querySelector(".startMenu__modal");
+
+startBtn.addEventListener("click", () => {
+  if (menuDisplay.style.display === "flex") {
```

Comment:
Rather than adding and removing a specific file, create a css class that has display set to flex or none and add/remove that css class